### PR TITLE
Add windows support for virtual_nic.with_stress

### PR DESCRIPTION
**Pull request by kimi1978wy**
Ping host when running heavyload stress tool

Signed-off-by: Yu Wang <wyu@redhat.com>

id:1737714

### CODE REVIEW

**Comment by kimi1978wy** [comment]
(1/3) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.virtual_nic.with_stress.q35: PASS (2382.81 s)
 (2/3) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.virtual_nic.with_stress.q35: PASS (2234.43 s)
 (3/3) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win7.x86_64.sp1.io-github-autotest-qemu.virtual_nic.with_stress.q35: PASS (2266.60 s)

File: qemu/tests/virtual_nic_stress.py [hunk 1]
```diff
@@ -31,9 +32,15 @@ def flood_ping(session, host_ip, os_type="linux"):
         logging.info("Flood ping for %s minutes" % flood_minutes)
         # TODO: windows guest doesn't have proper flood ping like linux,
         # will consider how to implement it for windows later
```
Comment (pezhang) [inline qemu/tests/virtual_nic_stress.py]: Perhaps "TODO" comment here can also be deleted like above in cfg?

Comment (kimi1978wy) [reply qemu/tests/virtual_nic_stress.py]: Yes, modified, thanks for reminder

**Comment by pezhang** [comment]
LGTM.
Acked by: Pei Zhang pezhang@redhat.com

**Comment by vivianQizhu** [comment]
ACK

**Pull request merged**